rootfs/etc/profile.d/set_cluster.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Usage:
+#   set_cluster <cluster-short-name>|off
+#
+#   With <cluster-short-name> updates the kubecfg file for the cluster with that short name (e.g. "corp")
+#   and updates KUBECONFIG to point ot that file.
+
+#   With "off", deletes the currently active kubecfg file and unsets KUBECONFIG
+#
+
+function set_cluster() {
```

Comment:
```suggestion
function set-cluster() {
```

Reply:
there's no prior art in geodesic of calling commands with `_`

Reply:
Standard is all shell functions use underscores because `-` is not a valid `bash` identifier character. This has to be a shell function in order to manipulate the caller's environment.

Reply:
We use underscores in all the shell function names. I didn't even realize dashes were allowed (they are not allowed in variable names).